feature/qos/otg_tests/ingress_police_mac_vlan/README.md

```diff
@@ -0,0 +1,181 @@
+# TE-18.4 Classify traffic on input using destination mac and police using 1 rate, 2 color marker
```

Comment:
I have a basic question: what is the purpose of destination-mac matching if this is a layer 3 service (this is not explicitly specified, but i'm assuming it is L3 based on the traffic flows and other TE-x.x test case descriptions).

Reply:
It's true this is an L3 interface use case.  The intent is to match all packs received on the interface.  There's no explicit classifier condition for matching all packets on an interface today in OC, so I suggested mac address.

We could introduce some new container for this in `/qos/classifiers/classifier/terms/term/conditions` if we think this is better.

Reply:
May I kindly request Sergey Fomin,
+1 that this is a L3 interface use case. 
Currently vendors like cisco support default policiers on sub-interface with can match on destination mac-address. In order to support similar construct seems this pull request would be helpful for the community.

Reply:
IMO using the interface/routing mac for wildcard match all is not quite right.

mac-criteria is often used with l2 services where it has very specific meaning/implementation, and mixing it with something completely different is suboptimal. 

Another question is,
>There's no explicit classifier condition for matching all packets on an interface today in OC, so I suggested mac address.

Does it have to be explicit? What if we assume implicit "match all" when no conditions are specified?

Reply:
Thanks Sergey Fomin, I agree mac-criteria is often used with l2 services.
Thinking aloud on how to match all packets with lack of classifier condition.

```
Does it have to be explicit? What if we assume implicit "match all" when no conditions are specified?
```

Great point let me check in this direction, thanks for your kind direction.

Reply:
> > There's no explicit classifier condition for matching all packets on an interface today in OC, so I suggested mac address.
> 
> Does it have to be explicit? What if we assume implicit "match all" when no conditions are specified?

That sounds good enough for this pull request.  @vishnureddybadveli can you update this PR remove the match critiera and add text to the readme  that we expect the lack of match conditions to cause all packets to be matched.  

Personally I like to avoid implicit rules like this, but we can take it up in a separate PR for the OC models and see where that goes.

Reply:
Thanks Darren

Added the required text.